hal/armv7m/imxrt/10xx/105x/peripherals.h

```diff
@@ -223,15 +225,19 @@
 
 /* FLASH */
 
-#define FLASH_NO                  FLASH_FLEXSPI1_MOUNTED + FLASH_FLEXSPI2_MOUNTED
+#define FLASH_NO                  FLEXSPI_COUNT
 #define FLASH_DEFAULT_SECTOR_SIZE 0x1000
 
-#define FLASH_FLEXSPI1_MOUNTED   1
-#define FLASH_FLEXSPI1           0x60000000
-#define FLASH_SIZE_FLEXSPI1      0x10000000
-#define FLASH_FLEXSPI1_INSTANCE  0x0
-#define FLASH_FLEXSPI1_QSPI_FREQ 0xc0000008
+#ifndef FLEXSPI_COUNT
+#define FLEXSPI_COUNT 1
+#endif
 
-#define FLASH_FLEXSPI2_MOUNTED 0
+#ifndef FLEXSPI1_PORT_MASK
+#define FLEXSPI1_PORT_MASK flexspi_slBusA1
+#endif
+
+#ifndef FLEXSPI2_PORT_MASK
+#define FLEXSPI2_PORT_MASK flexspi_slBusA1
```

Comment:
On `imxrt-105x` there is only single FlexSPI instance. Add appropriate precompiler `#error` in case if someone defines more than one instance.